src/andromede/simulation/output_values.py

```diff
@@ -43,18 +41,25 @@ class Variable:
         _name: str
         _value: Dict[TimeScenarioIndex, float] = field(init=False, default_factory=dict)
         _size: Tuple[int, int] = field(init=False, default=(0, 0))
+        ignore: bool = field(default=False, init=False)
 
         def __eq__(self, other: object) -> bool:
             if not isinstance(other, OutputValues.Variable):
                 return NotImplemented
-            return (
+            return (self.ignore or other.ignore) or (
                 self._name == other._name
                 and self._size == other._size
-                and self._value == other._value
+                and self._value.keys() == other._value.keys()
+                and all(
+                    math.isclose(self._value[key], other._value[key])
```

Comment:
Expose the parameters `abs_tol` and `rel_tol` so that we can tune them

Reply:
Done